admin/sql/updates/2018-06-25-drop-artist-credit-redirect.sql

```diff
@@ -0,0 +1,5 @@
+BEGIN;
+
+DROP TABLE IF EXISTS artist_credit_redirect CASCADE;
```

Comment:
Just move this to the other update script, keeping multiple schema change scripts in one PR doesn't really make sense.